src/PockyBot.NET/PockyBot.cs

```diff
@@ -31,7 +35,12 @@ public void Respond(Message message)
                 responder = _triggers.FirstOrDefault(x => _triggerResponseTester.ShouldTriggerInDirectMessage(message, x));
```

Comment:
From Craig:

Just reading this bit I don't know what it is supposed to do.

Reply:
We're getting the first trigger in the list that should trigger. Is there something you would like me to do to make this more clear?

Reply:
Ideally I'm thinking more of a event model. Like send the "MessageRecieved" event and have subscribers react to these events instead of linking them together through triggers